app/views/publications/monthly_statistics/show.html.erb

```diff
@@ -327,5 +330,15 @@
     <p class="govuk-body">
       Deferred candidates are counted in the academic year in which they start their course after training providers have re-confirmed their deferred offer. Providers need to do this before candidates intend to start their training.
     </p>
+
+    <p class="govuk-body">
+      Statistical releases from previous months during ITT2022 can be found here:
+      <%= link_to 'https://www.gov.uk/government/publications/monthly-statistics-on-initial-teacher-training-itt-recruitment' %>.
```

Comment:
this renders as a link to the current page. Is that right?

Reply:
it is not! I'll fi — thanks